sdk/servicebus/service-bus/samples/javascript/browseMessages.js

```diff
@@ -25,9 +25,10 @@ const queueName = process.env.QUEUE_NAME || "<queue name>";
 async function main() {
   const sbClient = new ServiceBusClient(connectionString);
   // If receiving from a subscription you can use the createReceiver(topic, subscription) overload
-  // Since browsing messages doesn't take a lock on the message, the receive mode passed to getReceiver
-  // is irrelevant to this sample code.
-  const queueReceiver = sbClient.createReceiver(queueName, "receiveAndDelete");
+  // Since browsing messages doesn't take a lock on the message, the receive mode passed to createReceiver
+  // is irrelevant.
```

Comment:
Perhaps we can remove this altogether now that mode is not a mandatory argument? I remember we added this so that people dont get confused as to what the mode does since we are showing that in the same?

Reply:
Yes.